psi4/share/psi4/fsapt/fsaptdiff.py

```diff
@@ -57,62 +57,62 @@
 
     fh = open('%s/fsapt.dat' % dirA, 'w')
     fh, sys.stdout = sys.stdout, fh
-    print '  ==> F-ISAPT: Links by Charge <==\n'
+    print('  ==> F-ISAPT: Links by Charge <==\n')
     stuffA = computeFsapt(dirA, False)
-    print '   => Full Analysis <=\n'
+    print('   => Full Analysis <=\n')
     printOrder2(stuffA['order2'], stuffA['fragkeys']) 
-    print '   => Reduced Analysis <=\n'
+    print('   => Reduced Analysis <=\n')
     printOrder2(stuffA['order2r'], stuffA['fragkeysr']) 
```

Comment:
Unfortunately, Libs, then Interp is against CMake's principles (https://cmake.org/cmake/help/v3.5/module/FindPythonInterp.html). And it explicitly states so few that I feel they should be followed :-)

That's why I was hoping to offload Py to pybind11. This may also be of interest (https://github.com/jkhoogland/FindPythonAnaconda/blob/master/FindPythonAnaconda.cmake).